File: 연결페이지/fetch_ref.py
import urllib.request
import re

url = "https://b22036tdchanmacgroup-source.github.io/HM_MISystem/"
try:
    response = urllib.request.urlopen(url)
    html = response.read().decode('utf-8')
    print("HTML Length:", len(html))
    
    # Find script src
    match = re.search(r'src="([^"]+)"', html)
    if match:
        print("Found src:", match.group(1))
        js_url = url + match.group(1).lstrip('/')
        print("JS URL:", js_url)
        js_response = urllib.request.urlopen(js_url)
        js_content = js_response.read().decode('utf-8')
        print("JS Length:", len(js_content))
        with open("C:/Users/user/.gemini/antigravity/scratch/ref_js.js", "w", encoding="utf-8") as f:
            f.write(js_content)
        print("Saved JS file.")
    else:
        print("No script src found.")
except Exception as e:
    print("Error:", e)

File: 연결페이지/fetch_ref.js
const https = require('https');
const fs = require('fs');

const rootUrl = "https://b22036tdchanmacgroup-source.github.io";
const jsUrl = rootUrl + "/HM_MISystem/assets/index-D-ES2Vm_.js";

console.log("Fetching JS from:", jsUrl);

https.get(jsUrl, (res) => {
  let data = '';
  res.on('data', (chunk) => { data += chunk; });
  res.on('end', () => {
    console.log("JS Length:", data.length);
    if (data.includes("<!DOCTYPE html>")) {
      console.log("Oops, got HTML instead of JS again. Let's see status:", res.statusCode);
    } else {
      const dest = "C:/Users/user/.gemini/antigravity/scratch/ref_js.js";
      fs.writeFileSync(dest, data);
      console.log("Saved JS file to:", dest);
    }
  });
}).on('error', (err) => {
  console.error("Error:", err);
});
